contrib/lib/src/serve.rs

```diff
@@ -104,6 +104,9 @@ impl Options {
     /// directory, rather than its parent. This is _not_ enabled by default.
     pub const NormalizeDirs: Options = Options(0b0100);
 
+    /// `Options` enabling caching based on the `Last-Modified` header.
```

Comment:
This should mention the `304` response with `If-Modified-Since`, maybe in another sentence or two.

Reply:
Done